tests/test_tab.py

```diff
@@ -0,0 +1,67 @@
+"""
+ Pylibui test suite.
+
+"""
+
+from pylibui.controls import Tab, Button
+from tests.utils import WindowTestCase
+
+
+class TabTest(WindowTestCase):
+
+    def setUp(self):
+        super().setUp()
+        self.tab = Tab()
+
+    def test_can_append_pages_to_tab(self):
+        """Tests that we can append pages to tab."""
```

Comment:
Actually, I really wanted to test that I can append pages to tab; however the only way to check the feature was using `getNumPages`.
I agree it is a bit misleading, I'll try to improve this :-)